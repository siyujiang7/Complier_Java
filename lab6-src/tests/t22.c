long even(long x){
  if (x == 0)
    return 1;
  return odd(x - 1);
}

long odd(long x){
  if (x == 0)
    return 0;
  return even(x - 1);
}

void main() {
  printf("200 is even?: ");
  printf("%d\n", even(200));

  printf("201 is even?: ");
  printf("%d\n", even(201));

}
