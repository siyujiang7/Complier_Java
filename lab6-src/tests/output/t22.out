200 is even?: 1
201 is even?: 0
